lstchain/scripts/lstchain_dl1ab.py

```diff
@@ -66,6 +69,12 @@
     help='key for the table of new parameters',
 )
 # Optional arguments
+parser.add_argument(
```

Comment:
add `type=Path` here, then it is directly a `Path` instance and you don't need to convert later.

Reply:
Done